paddle/scripts/paddle_build.sh

```diff
@@ -1597,6 +1611,42 @@ set -x
     fi   
 }
 
+function parallel_test_base_npu() {
+    mkdir -p ${PADDLE_ROOT}/build
+    cd ${PADDLE_ROOT}/build
+    if [ ${WITH_TESTING:-ON} == "ON" ] ; then
+    cat <<EOF
+    ========================================
+    Running unit npu tests ...
+    ========================================
+EOF
+
+set +x
+        ut_startTime_s=`date +%s`
+        test_cases=$(ctest -N -V | grep "op_npu" )        # cases list which would be run exclusively
```

Comment:
I think we should test all unittests inside `../npu` follder, not only the files ends with `op_npu`.

Reply:
DONE